assignments/arrays.js

```diff
@@ -63,27 +63,70 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+// console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+console.log(inventory[32].car_year, inventory[32].car_make, inventory[32].car_model );
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = function() {
+    let last = inventory.length -1;
+    return(inventory[last].car_make + " " + inventory[last].car_model);
+};
+console.log(lastCar());
+
+// var array1 = ['a', 'b', 'c'];
+
+// array1.forEach(function(element) {
+//   console.log(element);
+// });
+
+// expected output: "a"
+// expected output: "b"
+// expected output: "c"
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
-let carModels = [];
-console.log();
+let carModels = function () {
+    let sort = [];
+    for (let i = 0; i < inventory.length; i++) {
+        sort.push(inventory[i].car_model);        
+            
+        } 
+       return sort.sort();    
+    }
+    
+console.log(carModels());
 
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
-let carYears = [];
-console.log();
+let carYears = function(){
+    let years = [];
+    for (let i = 0; i < inventory.length; i++){
+        years.push(inventory[i].car_year);
+    }
+    return years;
+
+};
+console.log(carYears());
 
 // ==== Challenge 5 ====
 // The car lot manager needs to find out how many cars are older than the year 2000. Using the carYears array you just created, find out how many cars were made before the year 2000 by populating the array oldCars and logging it's length.
-let oldCars = [];
-console.log(); 
+let oldCars = [carYears()];
```

Comment:
`carYears()` is going to return an array, no need to place wrap `carYears()` in `[]`, it's producing the result like this
`[[1,2,3,4,5]]`. basically nesting the array inside an array, and in this problem we don't want that result, we want
`[1,2,3,4,5]`